content/en/blog/attribute-autocomplete-enhancement/index.md

```diff
@@ -0,0 +1,306 @@
+---
+title: "Enhancing Attribute Autocomplete Functionality -  Auto Correction and Type Suggestion"
+description: ""
+lead: "We discuss the two new enhancement added to the attribute name autocomplete feature in Texera, auto attribute correction and attribute type suggestion"
+date: 2023-06-18T20:58:20-07:00
+lastmod: 2023-06-18T20:58:20-07:00
+draft: true
+weight: 50
+#images: ["attribute-autocomplete-enhancement.jpg"]
+contributors: ["Chengxi Li"]
+---
+
+# Enhancing Attribute Autocomplete Functionality -  Auto Correction and Type Suggestion
+
+In this blog, we discuss the two new enhancement added to the attribute name autocomplete feature in Texera, auto attribute correction and attribute type suggestion, making the feature more user-friendly.
+
+## Background
+
+### What are operator properties
+
+Texera workflows consist of sequences of operators. Every operator has properties as the parameters of the data process function. For example, in Fig. 1, the "Filter" operator has three properties, `Attribute`, `Condition`, and `Value`. The property values in Fig. 1 indicate that the operator filters the records with amount greater than 1.
```

Comment:
Attribute properties are different from other properties. For example, in filter operator, it contains a property describe which attribute does this fitler apply on, and also contains other properties such as the condition of the fitler predicate. 

Make sure you describe this "Attribute AutoComplete" feature on the Attribute properties only.

Reply:
Added description about attribute properties.